docs/pages/beacon-management/networking.md

```diff
@@ -42,6 +42,10 @@ The primary purpose of ENRs is to facilitate node discovery and connectivity in
 
 Note that bootnodes are announced via ENR.
 
+Lodestar prints out its own ENR on startup, or it can alternatively be retrieved from the beacon node API by querying the [getNetworkIdentity](https://ethereum.github.io/beacon-APIs/#/Node/getNetworkIdentity) endpoint.
+
```

Comment:
How do you feel about adding an output sample here?  Or is that too much? 


suggestion: (not displaying the nested code block with github suggestion)

====================

The logs will show something similar to the following
```sh
info: discv5 worker started peerId=16Uiu.....t9LQ3, initialENR=enr:-Iu4QGE.........WRwgiMo, bindAddr4=/ip4/0.0.0.0/udp/9000

```
====================

Reply:
Was thinking about that too but I didn't want to include too many technical details here. So far people were able to spot it in the logs, there are not that many unless you have debug enabled